esp/esp/program/modules/handlers/commmodule.py

```diff
@@ -175,7 +181,10 @@ def commfinal(self, request, tl, one, two, module, extra, prog):
                                                       sendto_fn_name  = sendto_fn_name,
                                                       sender     = fromemail,
                                                       creator    = request.user,
-                                                      msgtext    = body,
+                                                      msgtext = unicode(loader.get_template('email/default_email_html.txt').render(DjangoContext(
```

Comment:
Likewise here.  Also, nit: can this either be on one line or be wrapped to a reasonable length?